bin/xtumlmc_build

```diff
@@ -809,7 +809,12 @@ sub xtumlmc_cleanse_inserts($$$)
       # If this element is one of the excluded, exclude it.
       foreach $subsystem_prefix (@_) {
         if ( /^INSERT INTO $subsystem_prefix/ ) {
-          while ( <$I> ) { if ( /^.*[;]\s*$/ ) { next MCJAVAOUTER; } }
+          if ( /.*PROXY.*/ ) {

```

Comment:
As I looked into this more closely, I see we have similar lines already in two other places (in functions just below where you edited).  I think the code being added should be changed to match our existing ones:

 if ( /^INSERT INTO .*_PROXY/ ) {
          while ( <$I> ) { if ( /^\s+'.*\.xtuml'\);/ ) { next MCJAVAOUTER; } }